server/src/main/java/io/crate/analyze/relations/AnalyzedRelation.java

```diff
@@ -66,7 +69,12 @@
      * </p>
      */
     @Nullable
-    Symbol getField(ColumnIdent column, Operation operation) throws UnsupportedOperationException;
+    default Symbol getField(ColumnIdent column, Operation operation) throws AmbiguousColumnException, ColumnUnknownException, UnknownObjectKeyExceptionalControlFlow, UnsupportedOperationException {
+        return getField(column, operation, true);
+    }
+
+    @Nullable
+    Symbol getField(ColumnIdent column, Operation operation, boolean errorOnUnknownObjectKey) throws AmbiguousColumnException, ColumnUnknownException, UnknownObjectKeyExceptionalControlFlow, UnsupportedOperationException;
```

Comment:
Could you extend the docs or so to clarify in which scenarios it is okay to use the shorter overload, vs. providing the `errorOnUnknownObjectKey` option?

Or rather - clarify in which case it is legitimate to rely on the `true` default?